ee/secureenclavesigner/secureenclavesigner.go

```diff
@@ -0,0 +1,197 @@
+//go:build darwin
+// +build darwin
+
+package secureenclavesigner
+
+import (
+	"context"
+	"crypto"
+	"crypto/ecdsa"
+	"encoding/base64"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"os/user"
+	"strings"
+	"time"
+
+	"github.com/kolide/krypto/pkg/echelper"
+	"github.com/vmihailenco/msgpack/v5"
+)
+
+type SignRequest struct {
```

Comment:
kinda weird that some are []bytes of b64 strings ... wonder if they should all be b64 string or all just straight []byte